book/4 - MultiWorlds/1 - Multiworlds Overview.html

```diff
@@ -0,0 +1,21 @@
+<h4>Multiple worlds</h4>
+
+<p>
+	Cuberite supports multiple worlds. Each world has its own `world.ini` file. Additional Worlds can be added by editing <code>server.ini</code>. See the example below.
```

Comment:
Also, Worlds should not be capital W I think, and `server.ini` should be `settings.ini`.